server/statistics/hot_peer_cache.go

```diff
@@ -454,3 +435,47 @@ func (f *hotPeerCache) updateHotPeerStat(newItem, oldItem *HotPeerStat, deltaLoa
 	}
 	return newItem
 }
+
+// TODO: remove it in future
+func (f *hotPeerCache) collectRegionMetrics(loads []float64, interval uint64) {
```

Comment:
Do you mind putting this function in the previous location? Changing the position brings unnecessary diff